src/methods/validators.rs

```diff
@@ -1,3 +1,64 @@
+//! Queries active validators on the network.
+//!
+//! Returns details and the state of validation on the blockchain.
+//!
+//! ## Examples
+//!
+//! - Get the validators for a specified epoch.
+//!
+//!     ```
+//!     use near_jsonrpc_client::{methods, JsonRpcClient};
+//!     use near_primitives::types::{EpochReference, EpochId, BlockReference, Finality};
+//!   
+//!     # #[tokio::main]
+//!     # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//!     let client = JsonRpcClient::connect("https://rpc.testnet.near.org");
+//!   
+//!     let block_request = methods::block::RpcBlockRequest {
+//!         block_reference: BlockReference::Finality(Finality::Final),
+//!     };
+//!     let block_response = client.call(block_request).await?;
+//!     let epoch_hash = block_response.header.epoch_id;
+//!   
+//!     let request = methods::validators::RpcValidatorRequest {
+//!         epoch_reference: EpochReference::EpochId(EpochId {
+//!             0: epoch_hash,
+//!         })
```

Comment:
Instead of making a `block` query first just to extract an epoch id, let's hard code the epoch id for the latest block and use an archival node.